### ISSUE TITLE
/metrics - invalid prometheus format (TYPE duplicates) /APPSERV-59

### ISSUE THREAD

**Issue by xadmi** [payara/Payara#4579]
The default **/metrics** endpoint returns duplicated TYPE entries which does not comply with Prometheus text-based exposition format.  
The specification under https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-details says:
> Only one TYPE line may exist for a given metric name.  

----------
This makes Prometheus Input Plugin for Telegraf to fail:
```
2020-03-18T14:05:30Z E! [inputs.prometheus] Error in plugin: error reading metrics for http://172.17.0.1:8080/metrics: reading text format failed: text format parsing error in line 40: second TYPE line for metric name "base_gc_time_total", or TYPE reported after samples
```
----------

## Expected Outcome

unique `# TYPE` lines under /metrics

## Current Outcome
```
# TYPE base_gc_time_total counter
# HELP base_gc_time_total Displays the approximate accumulated collection elapsed time in milliseconds. This attribute displays -1 if the collection elapsed time is undefined for this collector. The JVM implementation may use a high resolution timer to measure the elapsed time. This attribute may display the same value even if the collection count has been incremented if the collection elapsed time is very short.
base_gc_time_total{name="G1 Young Generation"} 1319
# TYPE base_gc_time_total counter
# HELP base_gc_time_total Displays the approximate accumulated collection elapsed time in milliseconds. This attribute displays -1 if the collection elapsed time is undefined for this collector. The JVM implementation may use a high resolution timer to measure the elapsed time. This attribute may display the same value even if the collection count has been incremented if the collection elapsed time is very short.
base_gc_time_total{name="G1 Old Generation"} 0
# TYPE base_gc_total_total counter
# HELP base_gc_total_total Displays the total number of collections that have occurred. This attribute lists -1 if the collection count is undefined for this collector.
base_gc_total_total{name="G1 Young Generation"} 26
# TYPE base_gc_total_total counter
# HELP base_gc_total_total Displays the total number of collections that have occurred. This attribute lists -1 if the collection count is undefined for this collector.
base_gc_total_total{name="G1 Old Generation"} 0
```

## Environment ##

- **Payara Version**: 5.201 (official payara/server-full docker image)
- **Edition**: Full

**Comment by fturizo** [comment]
@xadmi, thanks for reporting this issue. Our engieneers have already identified this issue which will be fixed with the upgrade to the next version of Eclipse MicroProfile Metrics (2.3). We have an internal issue to track this upgrade: `APPSERV-59`. Keep attention to the next public release to check if this new version of the library has been included.

**Comment by smillidge** [comment]
Fixed by #4582
